models/keyword.go

```diff
@@ -62,7 +68,7 @@ func FindKeywordById(id int64) (keyword *Keyword, err error) {
 
 func FindKeywordBy(Id int64, u *User) (*Keyword, error) {
```

Comment:
@junan Later on you can improve this by making this more loose coupling from the other model 😛

Reply:
It can be  done by passing just user `id`. I will improve in later ticket 👍